zulip/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [0.9.0] - 2023-05-26
+## Changed
+- Do not send heartbeat events to clients.
```

Comment:
Let's do the release in a separate commit; see fec8cc50c42f04c678a0318f60a780d55e8f382b for an example.

Reply:
(Or a separate PR -- we need to actually do the pypi upload too when we do it)